ui/views/tx/submit/result-codes-description.js

```diff
@@ -0,0 +1,70 @@
+const resultCodesDescription = {
+    tx_insufficient_fee: "The source account for transaction cannot pay the minimum fee.",
+    tx_insufficient_balance: "Fee would bring account below minimum balance.",
+    tx_bad_seq: "Sequence number does not match source account.",
+    tx_bad_auth: "Too few valid signatures / wrong network.",
+    tx_bad_auth_extra: "Unused signatures attached to transaction.",
+    tx_missing_operation: "No operation was specified.",
+    tx_not_supported: "The transaction type is not supported.",
+    tx_fee_bump_inner_failed: "The fee bump inner transaction failed.",
+    tx_too_early: "The transaction completes earlier than expected.",
+    tx_too_late: "The transaction completes later than expected.",
+    tx_no_source_account: "The source account was not found.",
+    tx_bad_minseq_age_or_gap: "Bad minimum sequence age or gap",
+    tx_bad_sponsorship: "The sponsorship is not confirmed",
```

Comment:
```suggestion
    tx_bad_sponsorship: "The sponsorship is not confirmed.",
```
It's either dots everywhere or nowhere.